doc/symbol.md

```diff
@@ -45,4 +45,14 @@ static Napi::Symbol Napi::Symbol::WellKnown(napi_env env, const std::string& nam
 Returns a `Napi::Symbol` representing a well-known `Symbol` from the
 `Symbol` registry.
 
+### For
+```cpp
+static Napi::Symbol Napi::Symbol::WellKnown(napi_env env, const std::string& name);
+```
+
```

Comment:
Please add the documentation for all the overloads of `Napi::Symbol Napi::Symbol::WellKnown()` methos.